utils/email.py

```diff
@@ -0,0 +1,52 @@
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.utils import timezone
+from smtpapi import SMTPAPIHeader
+import datetime
+
+def send_template_email(to_array, subject, template, context, sent_by=None):
+    body = render_to_string(template, context)
+    if sent_by != None:
+        email = EmailMultiAlternatives(
+            subject,
+            body,
+            sent_by,
+            to=to_array
```

Comment:
I think it's safe to say we only send emails with one recipient, so you can rename the parameter something like `to_email` and then do `to=[to_email],` in the call here